### DISCUSSION TITLE
Listing the objects directly from Object Url

### DISCUSSION THREAD

**Discussion by felixRajkumar** [aws/aws-sdk-java#2766] [Q&A] [answered]
Hi,
     Is there any possible way to list the objects in an s3 bucket using access key,  secret key and the URL which will be in the format https://bucketname.s3.region.amazonaws.com/${sub_folder}

**Comment by debora-ito** [comment, marked as answer]
You can call S3 ListObjectsV2 using "bucketname" as the bucket name and "sub_folder" as prefix.

https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjectsV2.html
